src/building/construction_building.c

```diff
@@ -330,7 +330,6 @@ int building_construction_place_building(building_type type, int x, int y)
     } else if (type == BUILDING_TRIUMPHAL_ARCH) {
         building_orientation = map_orientation_for_triumphal_arch(x, y);
     }
-    building_construction_offset_start_from_orientation(&x, &y, size);
```

Comment:
Isn't this needed for other building types?

Reply:
Wow, good catch. For some reason I thought that I had added that just for highways but nope, it's used for everything.